### PR TITLE
Update registry_config.j2 to fix BZ#1490738

### PR DESCRIPTION
**Pull request by jianlinliu**
Update registry_config.j2 to fix BZ#1490738

### CODE REVIEW

File: roles/openshift_hosted/templates/registry_config.j2 [hunk 1]
```diff
@@ -63,7 +63,7 @@ storage:
     keyfile: /etc/registry/gcs.json
 {%   endif -%}
 {%   if openshift_hosted_registry_storage_gcs_rootdirectory is defined %}
-    rootdirectory: {{ openshift_hosted_registry_storage_gcs_rootdirectory }}
+    rootdirectory: {{ openshift_hosted_registry_storage_gcs_rootdirectory | default('/registry') }}
```
Comment (sdodson) [inline roles/openshift_hosted/templates/registry_config.j2]: This change seems unrelated.

Comment (jianlinliu) [reply roles/openshift_hosted/templates/registry_config.j2]: Yeah, this is unrelated to swift issue, but it should another minor issue, if need open another PR to track gcs default rootdirectory issue, I will open another PR for that.

Comment (sdodson) [reply roles/openshift_hosted/templates/registry_config.j2]: no, this is fine
